tests/AppPsr7Test.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atk4\Ui\Tests;
+
+use Atk4\Core\Phpunit\TestCase;
+use Atk4\Ui\App;
+use Atk4\Ui\Exception\ExitApplicationError;
+use Atk4\Ui\Layout;
+
+class AppPsr7Test extends TestCase
+{
+    protected function getApp(): App
+    {
+        gc_collect_cycles();
```

Comment:
why is explicit GC here needed?

Reply:
probably a typo i will remove later and see if the test passes, but i see that some **first** tests fails.